pkgs/development/python-modules/pyquerylist/default.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  wheel,
+  tabulate,
+  coverage,
+  flake8,
+  nose2,
+}:
+buildPythonPackage rec {
+  pname = "pyquerylist";
+  version = "0-unstable-2024-10-21";
+
+  src = fetchFromGitHub {
+    owner = "markmuetz";
+    repo = pname;
+    rev = "5fab48f835cda4ed9462b5ad902726ba3d101554";
+    hash = "sha256-nwCcGNnKovY/P7tYyyivaMUn+nIYd0968/4cey9tqb8=";
+  };
+
+  propagatedBuildInputs = [
+    tabulate
+  ];
+
+  pythonImportsCheck = [ "pyquerylist" ];
+
+  checkInputs = [
+    coverage
+    flake8
+    nose2
+  ];
+
+  pyproject = true;
+  build-system = [
+    setuptools
+    wheel
+  ];
+
+  meta = {
+    description = "Extension of base Python list that you can query";
+    homepage = "https://github.com/markmuetz/pyquerylist";
```

Comment:
new packages are not allowed to be initialized without a maintainer. Please add yourself or find a willing adopter.